source/includes/_investors.md

```diff
@@ -1997,7 +1997,7 @@ Lists all transfers out for all investors.
 
 If you are migrating your existing investors over to Goji, then you need to onboard the existing investors using the dedicated [investor migration endpoint](#investors-post-platformapi-migrate) and include `migrationDetails` data.
 
-The `migrationDetails` allows the platform to provide an existing investor ID where applicable and provide the dilligence status as agreed with Goji before migration takes place. 
+The `migrationDetails` allows the platform to provide an existing investor ID where applicable and provide the diligence status as agreed with Goji before migration takes place. 
```

Comment:
I don't understand this.
I've copied this from the description above but why does it say `existing investor ID where applicable` if it's mandatory?